jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -173,52 +184,50 @@ pipeline {
                         terraform validate && terraform destroy -var-file user.tfvars --auto-approve
                     popd
                     '''
-                }
-
-           script {
-
-            // Jenkins Summary
-            clusterStatus = ""
-            if ( currentBuild.currentResult == "SUCCESS" ) {
-                MESSAGE = "CORTX Community Deploy is Success for the build ${build_id}"
-                ICON = "accept.gif"
-                STATUS = "SUCCESS"
-            } else if ( currentBuild.currentResult == "FAILURE" ) {
-                manager.buildFailure()
-                MESSAGE = "CORTX Community Deploy is Failed for the build ${build_id}"
-                ICON = "error.gif"
-                STATUS = "FAILURE"
+            }
 
+            script {
+                // Jenkins Summary
+                clusterStatus = ''
+                if ( currentBuild.currentResult == 'SUCCESS' ) {
+                    MESSAGE = "CORTX Community Deploy is Success for the build ${build_id}"
+                    ICON = 'accept.gif'
+                    STATUS = 'SUCCESS'
+            } else if ( currentBuild.currentResult == 'FAILURE' ) {
+                    manager.buildFailure()
+                    MESSAGE = "CORTX Community Deploy is Failed for the build ${build_id}"
+                    ICON = 'error.gif'
+                    STATUS = 'FAILURE'
             } else {
-                manager.buildUnstable()
-                MESSAGE = "CORTX Community Deploy Setup is Unstable"
-                ICON = "warning.gif"
-                STATUS = "UNSTABLE"
-            }
-            
-            clusterStatusHTML = "<pre>${clusterStatus}</pre>"
+                    manager.buildUnstable()
+                    MESSAGE = 'CORTX Community Deploy Setup is Unstable'
+                    ICON = 'warning.gif'
+                    STATUS = 'UNSTABLE'
+                }
 
-            manager.createSummary("${ICON}").appendText("<h3>CORTX Community Deploy ${currentBuild.currentResult} </h3><p>Please check <a href=\"${BUILD_URL}/console\">cluster setup logs</a> for more info <h4>Cluster Status:</h4>${clusterStatusHTML}", false, false, false, "red")
+                clusterStatusHTML = "<pre>${clusterStatus}</pre>"
```

Comment:
are we populating/getting `clusterStatus` variable from anywhere?

Reply:
code is already updated so check the code